geaflow/geaflow-dsl/geaflow-dsl-plan/src/test/java/com/antgroup/geaflow/dsl/udf/string/UDFStringTest.java

```diff
@@ -69,10 +70,48 @@ public void testBase64() {
     @Test
     public void testConcat() {
         String string = "ant group";
+        BinaryString binaryString = BinaryString.fromString(string);
         Concat test = new Concat();
         test.open(null);
-        assertEquals(test.eval(string, string, string),
+        assertEquals(test.eval(string, string, string), "ant groupant groupant group");
```

Comment:
Can we add a few tests related to null here？

Reply:
done!